README.md

```diff
@@ -105,6 +105,7 @@ The **M**inimum **S**upported **R**ust **V**ersions are:
   - This corresponds to the date that the `1.65.0` release was branched from `master`
 - `1.65.0` for Xtensa devices (**ESP32**, **ESP32-S2**, **ESP32-S3**)
 - `1.67.0` for all `async` examples (`embassy_hello_world`, `embassy_wait`, etc.)
+- latest `stable` when using the `defmt` feature
```

Comment:
Well, apart from my comment being a slight lie (pinned defmt means MSRV is 1.70 or so I believe), the feature is opt-in and probably shouldn't affect existing users who don't enable it. In case you want to rush the release, I'm not opposed to reverting though.